### DISCUSSION TITLE
Avis sur les type hits

### DISCUSSION THREAD

**Discussion by PierreS-alpha** [IPGP/autorino#2] [General]
Salut,

quel sont  vos avis sur les type hints? nécessaires? est-ce que ca n’alourdit pas inutilement le code, surtout pour les néophytes?

https://docs.python.org/3/library/typing.html
https://www.pythontutorial.net/python-basics/python-type-hints/

**Comment by sbonaime** [comment]
Bonjour,
Ca n'est pas nécessaire, mais ca permet de commenter dans le code et de rendre le code plus compréhensible par les collègues et peut être les débutants. 

Mais peut être mieux: Les dataclass qui combinent hints et __init__ (et même __post_init__) 

https://youtu.be/FcVCfGJrkUQ
https://docs.python.org/3/library/dataclasses.html
